src/ui/components/pages/Catalog/CatalogExplorer/CatalogExplorerCards/CatalogExplorerCards.tsx

```diff
@@ -21,14 +21,19 @@ export type Props = {
     className?: string;
     packages: {
         packageName: string;
+        version: string;
         packageIconUrl?: string;
         packageDescription: string;
         packageHomeUrl?: string;
         catalogId: string;
     }[];
     search: string;
     onSearchChange: (search: string) => void;
-    onRequestLaunch: (params: { packageName: string; catalogId: string }) => void;
+    onRequestLaunch: (params: {
+        packageName: string;
+        catalogId: string;
+        version: string;
```

Comment:
Be careful, this is a serious design mistake.  
A package is fully determined by `{packageName, catalogId}` it's the primary key of the table.  
The parent of of `<CatalogExplorerCards />` is able to resolve the version from `{packageName, catalogId}`.